doc/zapiska.tex

```diff
@@ -70,8 +71,31 @@ \section*{реферат}
 % ommited because it's only used for groupping
 % основну частину:
   \unnumberedSection{вступ}  
-  \lipsum[1-3]
-  \BgThispage
+  Задача, яку розглядає дана робота ---
+  переклад та перевірка точності перекладу текстів (наукових робіт)
+  з натуральної мови на формальну (УДК шифр).
+  
+  Точність УДК шифрів є ключовою для ефективної класифікації
+  та пошуку наукової літератури.
+  Однак, ручний вибір та перевірка УДК шифрів вимагає значних зусиль,
+  займає багато часу та супроводжується високим ризиком виникнення помилок.
+  Зовнішня перевірка може зменшити ризик помилок, але вимагає додаткових зусиль.
+  Натомість, автоматизоване рішення може значно зменшити час та ресурси,
+  необхідні для вибору та перевірки, а також мінімsізувати ризик помилок.
+  Тому ця робота має на меті розробити надійний
+  та ефективний автоматизований інструмент для вибору
+  та перевірки УДК шифрів наукових робіт.
+
+  Перевірка може бути виконана у два кроки:
+  \begin{enumerate}[labelindent=\parindent, leftmargin=*]
+    \item Переклад
+    \item Порівняння наданого шифру із перекладом.
+  \end{enumerate}
```

Comment:
Didn't find any info about list formatting in guidelines, so I made it to my taste
![image](https://user-images.githubusercontent.com/30831873/235770243-f845aa21-97e8-4d79-a9b7-a9643cfb6aba.png)

Reply:
UPD: I'll use the same indentation as paragraph for each list level for future lists, but this exact list is removed from the PR